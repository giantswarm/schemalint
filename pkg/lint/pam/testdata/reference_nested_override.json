{
  "properties": {
    "person": {
      "$ref": "nested.json",

      "properties": {
        "addresses": {
          "items": {
            "properties": {
              "city": {
                "title": "City - title override",
                "type": "string"
              }
            },
            "type": "object"
          },
          "type": "array"
        }
      },
      "type": "object"
    }
  },
  "type": "object"
}
