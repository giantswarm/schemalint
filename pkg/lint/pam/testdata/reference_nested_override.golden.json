{
  "/properties/person": {
    "Title": null,
    "Description": null,
    "Examples": null
  },
  "/properties/person/properties/addresses": {
    "Title": {
      "Value": "Addresses",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The person's addresses.",
      "ReferenceLevel": 1
    },
    "Examples": null
  },
  "/properties/person/properties/addresses/items/properties/city": {
    "Title": {
      "Value": "City - title override",
      "ReferenceLevel": 0
    },
    "Description": {
      "Value": "The city.",
      "ReferenceLevel": 1
    },
    "Examples": null
  },
  "/properties/person/properties/addresses/items/properties/state": {
    "Title": {
      "Value": "State",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The state.",
      "ReferenceLevel": 1
    },
    "Examples": null
  },
  "/properties/person/properties/addresses/items/properties/street": {
    "Title": {
      "Value": "Street",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The street address.",
      "ReferenceLevel": 1
    },
    "Examples": null
  },
  "/properties/person/properties/addresses/items/properties/zip": {
    "Title": {
      "Value": "Zip",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The zip code.",
      "ReferenceLevel": 1
    },
    "Examples": null
  },
  "/properties/person/properties/age": {
    "Title": {
      "Value": "Age",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "Age in years which must be equal to or greater than zero.",
      "ReferenceLevel": 1
    },
    "Examples": null
  },
  "/properties/person/properties/firstName": {
    "Title": {
      "Value": "First Name",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The person's first name.",
      "ReferenceLevel": 1
    },
    "Examples": null
  },
  "/properties/person/properties/lastName": {
    "Title": {
      "Value": "Last Name",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The person's last name.",
      "ReferenceLevel": 1
    },
    "Examples": null
  }
}
